src/ui/component/popover/popover.scss

```diff
@@ -0,0 +1,61 @@
+@import '@/ui/style/theme.scss';
+
+.okp4-dataverse-portal-popover-trigger-icon {
+  @include with-theme {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition: transform 200ms ease-out;
+    
+    &.flipped {
+      transform: rotate(180deg);
+    }
+
+    path {
+      fill: themed("text");
+      stroke: themed("text");
+    }
+  }
+}
+
+.okp4-dataverse-portal-popover-trigger-button {
```

Comment:
[Radix Popover](https://www.radix-ui.com/primitives/docs/components/popover) generates a `<button/>` element for the trigger by default. That's why `border: 0` is useful [here](https://github.com/okp4/dataverse-portal/pull/442#discussion_r1295897642). We can customize the Trigger element using [composition](https://www.radix-ui.com/primitives/docs/guides/composition) with `asChild`. In our case of a reusable component, that might be better to use a div, especially in case `trigger` element contains nested buttons, which would result in an invalid html tree.
We can update the component in that direction if you agree with, and thus rename `button` to `element`, as you suggested.

Reply:
as discussed together, we'll keep the basic Radix handling because `div` isn't a focusable element and would make us loose accessibility.